packages/odyssey-react-mui/src/labs/AppTile.tsx

```diff
@@ -30,12 +30,15 @@ import {
   DesignTokens,
   useOdysseyDesignTokens,
 } from ".././OdysseyDesignTokensContext";
-import { Heading5, Paragraph, Subordinate, Support } from ".././Typography";
+import { Heading5, Paragraph, Subordinate } from ".././Typography";
 import { Box } from ".././Box";
 
+export const APP_TILE_COMPACT_IMAGE_HEIGHT = "40px";
```

Comment:
Do we want these values to scale with font size changes? Or, are they fine being static values?